packages/core/src/account/types.ts

```diff
@@ -5,14 +5,21 @@ import type { z } from "zod";
 import type { SupportedTransports } from "../client/types";
 import type { SmartAccountSigner } from "../signer/types";
 import type { BatchUserOperationCallData } from "../types";
-import type { BaseSmartAccountParamsSchema } from "./schema";
+import type {
+  BaseSmartAccountParamsSchema,
+  SimpleSmartAccountParamsSchema,
+} from "./schema";
 
 export type SignTypedDataParams = Omit<SignTypedDataParameters, "privateKey">;
 
 export type BaseSmartAccountParams<
   TTransport extends SupportedTransports = Transport
 > = z.infer<ReturnType<typeof BaseSmartAccountParamsSchema<TTransport>>>;
 
+export type SimpleSmartAccountParams<
+  TTransport extends SupportedTransports = Transport
+> = z.infer<ReturnType<typeof SimpleSmartAccountParamsSchema<TTransport>>>;
```

Comment:
left another comment about this somewhere else, but I think this is the same as `z.output<typeof ...>`